From 7498aa82c3a1d094dd4f0dd81cd76efb70ba5405 Mon Sep 17 00:00:00 2001
From: Dimitar Dimitrov <dimitar@dinux.eu>
Date: Thu, 10 Nov 2016 22:03:45 +0200
Subject: [PATCH 08/13] testsuite: Workaround stack underflow with PRU

PRU's CRT is so lean that even a few bytes underflow are caught by simulator.

gcc/testsuite/ChangeLog:

2018-04-15  Dimitar Dimitrov  <dimitar@dinux.eu>

	* gcc.c-torture/execute/pr47237.c: Skip for pru.

Signed-off-by: Dimitar Dimitrov <dimitar@dinux.eu>
---
 gcc/testsuite/gcc.c-torture/execute/pr47237.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/testsuite/gcc.c-torture/execute/pr47237.c b/gcc/testsuite/gcc.c-torture/execute/pr47237.c
index 98124065b2f..03a40e8fd9a 100644
--- a/gcc/testsuite/gcc.c-torture/execute/pr47237.c
+++ b/gcc/testsuite/gcc.c-torture/execute/pr47237.c
@@ -1,4 +1,4 @@
-/* { dg-xfail-if "can cause stack underflow" { nios2-*-* } } */
+/* { dg-xfail-if "can cause stack underflow" { nios2-*-* pru*-*-*} } */
 /* { dg-require-effective-target untyped_assembly } */
 #define INTEGER_ARG  5
 
-- 
2.11.0

